onedrive_service/src/tests/test_service.py

```diff
@@ -0,0 +1,38 @@
+"""Test OneDriveService functionality."""
+# pylint: disable=protected-access
+
+import unittest
+
+import mock
+
+from onedrive_service.service import OneDriveService
+
+
+class TestOneDriveService(unittest.TestCase):
+    """
+    Test OneDriveService functionality
+    """
```

Comment:
I would add tests for other methods of the client.

Reply:
not yet. It will be in a separate request.